### DISCUSSION TITLE
Question：Mismatch of epoch's number btween training and validation when using CSVLogger

### DISCUSSION THREAD

**Discussion by ZJZhang123** [Lightning-AI/pytorch-lightning#13469] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Hi Community,
I am trying to log metrics at the each epoch's end. However, there is something strange happened: as the figure shown, the epoch's number of validation log isn't consistent with the counterpart of training log. Besides, it seems that the val log is followed by training log.
![image](https://user-images.githubusercontent.com/71306074/176673379-487b823d-5c0b-4572-a0ea-9536ec7d4c86.png)

Here is my code:
![image](https://user-images.githubusercontent.com/71306074/176673637-c50a7810-985c-4097-b0a1-a16082878715.png)

**Comment by akihironitta** [comment]
@ZJZ1223 Looks similar to #11605. Would you mind having a look at the linked issue?
